lib/webmushra/pages/PreferenceTestPage.js

```diff
@@ -0,0 +1,184 @@
+/*************************************************************************
+         (C) Copyright AudioLabs 2017 
+
+This source code is protected by copyright law and international treaties. This source code is made available to You subject to the terms and conditions of the Software License for the webMUSHRA.js Software. Said terms and conditions have been made available to You prior to Your download of this source code. By downloading this source code You agree to be bound by the above mentionend terms and conditions, which can also be found here: https://www.audiolabs-erlangen.de/resources/webMUSHRA. Any unauthorised use of this source code may result in severe civil and criminal penalties, and will be prosecuted to the maximum extent possible under law. 
+
+**************************************************************************/
+
+function PreferenceTestPage(_pageManager, _pageTemplateRenderer, _audioContext, _bufferSize, _audioFileLoader, _stimuli,  _session, _pageConfig, _errorHandler, _language) {
+  this.pageManager = _pageManager;
+  this.pageTemplateRenderer = _pageTemplateRenderer;
+  this.audioContext = _audioContext;
+  this.bufferSize = _bufferSize;
+  this.audioFileLoader = _audioFileLoader;
+  this.session = _session;
+  this.pageConfig = _pageConfig;
+  this.errorHandler = _errorHandler;
+  this.language = _language;
+  this.div = null;
+  this.fpc = null;
+
+  this.currentItem = null; 
+
+  this.stimuli = _stimuli;
+	this.played = [];
+  for (var i = 0; i < this.stimuli.length; ++i) {
+    this.audioFileLoader.addFile(this.stimuli[i].getFilepath(), (function (_buffer, _stimulus) { _stimulus.setAudioBuffer(_buffer); }), this.stimuli[i]);
+    this.played.push(false);
+  }
+
+  this.filePlayer = null;
+  this.choice = null;
+  this.time = 0; 
+  this.startTimeOnPage = null;
+}
+
+
+
+PreferenceTestPage.prototype.getName = function () {
+  return this.pageConfig.name;
+};
+
+PreferenceTestPage.prototype.init = function () { 
+  this.filePlayer = new FilePlayer(this.audioContext, this.bufferSize, this.stimuli, this.errorHandler, this.language, this.pageManager.getLocalizer());
+  
+  if (typeof this.pageConfig.mustPlayback !== "undefined") {
+    this.filePlayer.genericAudioControl.addEventListener((function (_event) {
+      if (_event.name == this.pageConfig.mustPlayback) {
+        this.played[_event.index] = true;
+        if (this.played.every(element => element === true)){
```

Comment:
Not sure, but I think the use of `=>` is incorrect here...

Reply:
```suggestion
        if (this.played.every((element) => element === true)){
```

Reply:
I don't think this code was written by me. I'm happy to make the change, if you want me to, though. Please confirm.

Reply:
Yes, go a head. I discovered this using the `grunt package` task. You can check yourself if this solves the error.

I can try to enable this on github action in the future

Reply:
@nullpunktTUD i realized this is ES6 syntax, are you able to find and alternative that works on older browsers?

Reply:
I'm afraid I am not experienced enough with JS to fix this with confidence.

Reply:
Ok. I will look into this later

Reply:
Can you find an alternative for this?

Reply:
@Simon-Stone do you have an update on this? i'm not a js expert myself but maybe chatgpt could help here?

Reply:
I don't have an update, no. I can try to petition our AI overlords, though, and report back